Cargo.toml

```
@@ -104,7 +104,7 @@ etcd-client = "0.12"
 fst = "0.4.7"
 futures = "0.3"
 futures-util = "0.3"
-greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "06f6297ff3cab578a1589741b504342fbad70453" }
+greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "86e143918b70fe8c7b11df0f2cfcea76934f6217" }
```

This commit is on your forked branch. Should use the one in main instead:

```suggestion
greptime-proto = { git = "https://github.com/GreptimeTeam/greptime-proto.git", rev = "1bd2398b686e5ac6c1eef6daf615867ce27f75c1" }
```

---

Should we use `branch` instead of `rev`?

---

Updated

---

Depending on the branch may lead to unexpected updates (since the lock file is usually ignored in review).